styles/email/_email-base.scss

```diff
@@ -0,0 +1,134 @@
+//  email.less
+//  Styles contained in the _email-base.scss will be output in the email.less file
+//  and will be included in a <style> tag in emails. Non-inline styles should be used for styles that can't be applied
+//  as inline CSS, such as media queries.
+
+//
+//  Variables
+//  _____________________________________________
+
+$font-size-unit-convert: false; // Prevents font-related measurements from being converted to the incompatible rem unit
+
+//
+//  Resets
+//  ---------------------------------------------
+
+//  Client-specific resets (from the Salted email template: https://github.com/rodriguezcommaj/salted)
+
+
+//  Prevent WebKit and Windows mobile from changing default text sizes
+body,
+table,
+td,
+a {
+    -ms-text-size-adjust: 100%; // sass-lint:disable-line no-vendor-prefixes
+    -webkit-text-size-adjust: 100%; // sass-lint:disable-line no-vendor-prefixes
+}
+
+//  Allow smoother rendering of resized images in Internet Explorer (such as high-resolution logo)
+img {
+    -ms-interpolation-mode: bicubic; // sass-lint:disable-line no-misspelled-properties no-vendor-prefixes
+}
+
+//  Remove spacing between tables in Outlook 2007 and up
+// sass-lint:disable-all
+table,
+td {
+    mso-table-lspace: 0pt;
+    mso-table-rspace: 0pt;
+}
+// sass-lint:enable-all
```

Comment:
Ignore specific rule, for the specific line, not everything for a block of code.